recipe/meta.yaml

```diff
@@ -48,7 +48,6 @@ test:
     - pytest
     - pytest-xdist
     - mock  # [py27]
-  run_test.sh
```

Comment:
👍 Yeah this should never be needed as it will be copied over to the test directory and run in the test environment automatically. It amounts to something like this plus whatever the user adds.

```yaml
test:
  files:
    - run_test.sh
    ...
  commands:
    - bash run_test.sh
    ...
  ...
```